src/modules/core/components/ActionsList/ActionsListItem.css

```diff
@@ -0,0 +1,134 @@
+/*
+ * @NOTE Unique values
+ */
+@value mainTextSize: 13px;
+@value popoverWidth: 400px;
+@value popoverDistance: 10px;
+
+.main {
+  display: flex;
+  margin-bottom: 1px;
+  padding: 26px 20px;
+  max-height: 89px;
+  border-radius: var(--radius-large);
+  background-color: var(--colony-white);
+  font-size: mainTextSize;
+  font-weight: var(--weight-bold);
+  color: var(--dark);
+  letter-spacing: var(--spacing-medium);
+  box-shadow: 0px 2px 11px var(--drop-shadow);
+  cursor: pointer;
+  outline: 0;
+}
+
+.main:hover {
+  background-color: var(--temp-grey-blue-8);
+}
+
+.avatar {
+  margin-top: -2px;
+  padding-right: 15px;
+  cursor: pointer;
+}
+
+.avatar figure {
+  height: 41px;
+  width: 41px;
+  border: 2px solid transparent;
+  border-radius: 100%;
+}
+
+.avatar:hover figure {
+  border-color: var(--primary);
+}
+
+.status::before {
+  display: block;
+  margin: -22px 0 0 -17px;
+  height: 81px;
+  width: 4px;
+  position: absolute;
+  border-radius: var(--radius-large);
+  content: '';
+}
+
+.stateRed {
+  composes: status;
+}
+
+.stateRed::before {
+  background-color: var(--pink);
+}
+
+.stateBlue {
+  composes: status;
+}
+
+.stateBlue::before {
+  background-color: var(--colony-blue);
+}
+
+.stateYellow {
+  composes: status;
+}
+
+.stateYellow::before {
+  background-color: var(--golden);
+}
+
+.content {
+  flex: 1 1 100%;
+  min-width: 0;
```

Comment:
What is the reason behind setting min-width to 0? 🤔

Reply:
This is minor "hack" in order to get text ellipsis to work properly.

You are right I should have left a comment explaining this here. And I will :)